packages/next/client/image.tsx

```diff
@@ -50,7 +50,7 @@ type PlaceholderValue = 'blur' | 'empty'
 
 type ImgElementStyle = NonNullable<JSX.IntrinsicElements['img']['style']>
 
-interface StaticImageData {
+export interface StaticImageData {
```

Comment:
This is exported but not used anywhere.

Were you going to import it in `./packages/next/types/global.d.ts`?

Reply:
Initially it was yeah but it causes issues during dev from not being available until after a build so this export can be removed